resilience4j-framework-common/src/main/java/io/github/resilience4j/common/ratelimiter/configuration/CommonRateLimiterConfigurationProperties.java

```diff
@@ -167,6 +167,10 @@ public Integer getLimitForPeriod() {
          * @param limitForPeriod the permissions limit for refresh period
          */
         public InstanceProperties setLimitForPeriod(Integer limitForPeriod) {
+            if (limitForPeriod < 1) {
```

Comment:
This is second validation added. To avoid negative and zero permits in Semaphore